dataparser/data_queue/scanner.py

```diff
@@ -4,9 +4,9 @@
 import os
 import xml.etree.ElementTree as ET
 from robot.errors import DataError
-from finder import finder
+from .finder import finder
 from data_parser.data_parser import DataParser
-from queue import ParsingQueue
+from .queue import ParsingQueue

```

Comment:
`data_queue` ?

Reply:
I've just changed the folder name. The file, queue.py, itself stays the same